code/datums/diseases/advance/symptoms/genetics.dm

```diff
@@ -29,7 +29,7 @@ Bonus
 
 /datum/symptom/genetic_mutation/Activate(datum/disease/advance/A)
 	..()
-	if(prob(SYMPTOM_ACTIVATION_PROB * 5)) // 15% chance
+	if(prob(SYMPTOM_ACTIVATION_PROB * 3)) // 15% chance

```

Comment:
Yeah, i increased SYMPTOM_ACTIVATION_PROB to 5% to make up for the loss of stacking.